go-controller/pkg/node/default_node_network_controller.go

```diff
@@ -443,28 +444,99 @@ func handleNetdevResources(resourceName string) (string, error) {
 	return netdevice, nil
 }
 
-func createNodeManagementPorts(name string, nodeAnnotator kube.Annotator, waiter *startupWaiter,
-	subnets []*net.IPNet) ([]managementPortEntry, *managementPortConfig, error) {
-	// If netdevice name is not provided in the full mode then management port backed by OVS internal port.
-	// If it is provided then it is backed by VF or SF and need to determine its representor name to plug
-	// into OVS integrational bridge
+func exportManagementPortAnnotation(netdevName string, nodeAnnotator kube.Annotator) error {
+	klog.Infof("Exporting management port annotation")
+
+	deviceID, err := util.GetDeviceIDFromNetdevice(netdevName)
+	if err != nil {
+		return err
+	}
+	vfindex, err := util.GetSriovnetOps().GetVfIndexByPciAddress(deviceID)
+	if err != nil {
+		return err
+	}
+	pfindex, err := util.GetSriovnetOps().GetPfIndexByVfPciAddress(deviceID)
+	if err != nil {
+		return err
+	}
+
+	return util.SetNodeManagementPortAnnotation(nodeAnnotator, pfindex, vfindex)
+}
+
+func importManagementPortAnnotation(node *kapi.Node) (string, error) {
+	klog.Infof("Import management port annotation")
+	pfId, vfId, err := util.ParseNodeManagementPortAnnotation(node)
+
+	if err != nil {
+		return "", err
+	}
+
+	return sriovnet.GetVfRepresentorDPU(fmt.Sprintf("%d", pfId), fmt.Sprintf("%d", vfId))
```

Comment:
This one actually should be `util.GetSriovnetOps().GetVfRepresentorDPU()`

Reply:
Even though I have https://github.com/k8snetworkplumbingwg/sriovnet/pull/64 pending, I will fix this here too. After that PR is merged, I will clean up this code too to use that new function.